src/frontend/src/eth/components/fee/FeeDisplay.svelte

```diff
@@ -46,38 +40,13 @@
 
 		clearTimeout(timer);
 	});
-
-	const balance =
-		nonNullish($feeTokenIdStore) && nonNullish($balancesStore)
-			? $balancesStore[$feeTokenIdStore]?.data ?? BigNumber.from(0n)
-			: undefined;
-
-	let insufficientFeeFunds = false;
-	$: insufficientFeeFunds = nonNullish(fee) && nonNullish(balance) && balance.lt(fee);
 </script>
 
 <label for="balance" class="font-bold px-4.5"
 	>Max fee <small>(likely in &lt; 30 seconds)</small>:</label
 >
-<div id="balance" class="font-normal px-4.5 mb-4 break-all min-h-10">
-	{#if nonNullish(fee)}
-		<div transition:fade>
-			{formatToken({
-				value: fee,
-				displayDecimals: EIGHT_DECIMALS
-			})}
-			{$feeSymbolStore ?? ''}
-			{#if insufficientFeeFunds && nonNullish(balance)}
-				<p transition:slide={{ duration: 250 }} class="text-cyclamen text-xs">
-					{replacePlaceholders($i18n.send.assertion.not_enough_tokens_for_gas, {
-						$balance: formatToken({
-							value: balance,
-							displayDecimals: EIGHT_DECIMALS
-						}),
-						$symbol: $feeSymbolStore ?? ''
-					})}
-				</p>
-			{/if}
-		</div>
+<div id="balance" class="font-normal px-4.5 mb-4 break-all min-h-6">
+	{#if nonNullish(fee) && nonNullish($feeSymbolStore) && nonNullish($feeTokenIdStore)}
+		<FeeAmountDisplay {fee} feeSymbol={$feeSymbolStore} feeTokenId={$feeTokenIdStore} />
```

Comment:
Not sure, instead of passing `feeSymbol` and `feeTokenId`, the component `FeeAmountDisplay` cannot retrieve those value from the `FeeContext`?

Reply:
It is used by `EthereumEstimatedFee` too, and in that case the values are passed differently. We can try and armonize the two flows

Reply:
Ah ok. Then let's keep it as implemented, thanks.